snippets/olmv1-known-issue-service-accounts.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/olmv1-installing-an-operator.adoc
+// * release_notes/ocp-4-17-release-notes.adoc (enterprise-4.17 branch only)
+
+:_mod-docs-content-type: SNIPPET
+
+There is a known issue in {olmv1}. If you do not assign the correct role-based access controls (RBAC) to an extension's service account, {olmv1} gets stuck and reconciliation stops.
+
+Currently, {olmv1} does not have tools to help extension administrators find the correct RBAC for a service account.
+
+Because {olmv1} is a Technology Preview feature and must not be used on production clusters, you can avoid this issue by using the more permissive RBAC included in the documentation.
```

Comment:
@michaelryanpeter because it is TP and we have no tool to help customer to get correct RBAC, you mention `you can avoid this issue by using the more permissive RBAC included in the documentation`. is it https://github.com/openshift/openshift-docs/pull/82245/files#diff-e44e70381349dd00e73c99c547be95941927faeaf5958930c49b641c4db8c02dR77-R84? thanks

Reply:
Yes, @kuiwang02 . That is the more permissive RBAC. Would you like me to include it in the snippet?

Reply:
for me, it is not necessary because the example provide such RBAC.